public/views/FormView.js

```diff
@@ -0,0 +1,9 @@
+var Backbone = require('backbone');
+
+module.exports = Backbone.View.extend({
+  el: '#add-plant-form',
+  render: function() {
+    $(this.el).html('<div><input name="name" type="text" placeholder="Plant name"/></div><div><input name="number" type="text" placeholder="Number of plants"/></div><div><input name="spaceNeeded" type="text" placeholder="Space per plant (in sq. ft.)"/></div><div><p>Date to plant</p><input name="datePlanted" type="date" placeholder="Date to plant"/></div><div><p>Date to harvest</p><input name="dateHarvested" type="date" placeholder="Date to harvest"/></div><div><input type="submit" value="Plant it!"></div>')
+    return this;
+  }
+});
```

Comment:
Having a FormView is a good idea, but that means you want to handle submit events in your form. Right now, the form in your markup doesn't have an action, so it won't try to post to a server endpoint, and since there's no submit handler, it won't gather any data to put into a collection on submit here either. To do that here you would need first some event handlers (this is where you'd use an events hash in your view):

```
module.exports = Backbone.View.extend({ 
   events: {
     'submit': 'handleSubmit'
   },
   handleSubmit: function (ev) {
     ev.preventDefault();
     // gather form inputs and add them to your collection using the create method
     // which means when this view is instantiated, you need to pass it a collection reference
});
```